pkg/cloudprovider/aws/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -107,6 +112,23 @@ func (e EKS) nodeLabelArg() string {
 	return fmt.Sprintf("%s%s", nodeLabelArg, strings.Join(labelStrings, ","))
 }
 
+// systemReservedArg gets the kubelet-defined arguments for any valid resource
+// values that are specified within the system reserved resource list
+func (e EKS) systemReservedArg() string {
+	var args []string
+	if e.KubeletConfig.SystemReserved != nil {
+		for _, name := range []v1.ResourceName{v1.ResourceCPU, v1.ResourceMemory, v1.ResourceEphemeralStorage} {
```

Comment:
Any reason to limit to these types? Could we pass whatever's present in the map?

Reply:
It's in the comment that you posted above. Only cpu, memory, and ephemeral storage are supported